rules-reviewed/hibernate/hibernate-xml.windup.xml

```diff
@@ -30,5 +32,22 @@
                 </iteration>
             </perform>
         </rule>
+	<rule id="hibernate-xml-02000">
+            <when>
+               <xmlfile as="1" matches="/hibernate-configuration" />
+               <xmlfile as="2" matches="//property[@name='hibernate.jdbc.use_streams_for_binary']" from="1" />
+            </when>
+            <perform>
+                <iteration over="2">
+                    <hint title="hibernate.jdbc.use_streams_for_binary must be set according to the provider" effort="0">
+                         <message><![CDATA[
+                        If you are using Oracle and using the ```materialized_clob``` or ```materialized_blob``` properties, this global environment variable must be set to true. 
```

Comment:
There is a link that detailly explains the problem. Is it needed to tell more information here?